qa/tasks/cephfs/filesystem.py

```diff
@@ -711,6 +711,35 @@ def create(self):
             if session_timeout != 60:
                 self.set_session_timeout(session_timeout)
 
+            log.debug(f"fs_config: {self.fs_config}")
+            if self.fs_config.get('subvols', None) is not None:
+                log.debug(f"Creating {self.fs_config.get('subvols')} subvols "
+                          "for filesystem '{self.name}'")
+                if not hasattr(self._ctx, "created_subvols"):
+                    self._ctx.created_subvols = dict()
+
+                subvols = self.fs_config.get('subvols')
+                assert(isinstance(subvols, dict))
+                assert(isinstance(subvols['create'], int))
+                assert(subvols['create'] > 0)
+
+                for sv in range(0, subvols['create']):
+                    sv_name = f'sv_{sv}'
+                    self.mon_manager.raw_cluster_cmd(
+                        'fs', 'subvolume', 'create', self.name, sv_name,
+                        self.fs_config.get('subvol_options', ''))
+
+                    if self.name not in self._ctx.created_subvols:
+                        self._ctx.created_subvols[self.name] = []
+                    
+                    subvol_path = self.mon_manager.raw_cluster_cmd(
+                        'fs', 'subvolume', 'getpath', self.name, sv_name)
+                    subvol_path = subvol_path.strip()
```

Comment:
Is this actually necessary? (It should not be!)

Reply:
stripping off the the trailing `'\n'` is necessary
I found this out when I looked up the teuthology logs and saw the dumped path which included the trailing `'\n'`

Reply:
Seems you found a bug then? I don't think it should be in the output.

Reply:
@batrick fyi - check `qa/tasks/cephfs/test_volumes.py`
every place where a `getpath` is requested, there's a strip operation following it